dev/examples/reduceMotion.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { motion, useReducedMotion } from "@framer"
+
+const style = {
+    width: 100,
+    height: 100,
+    background: "red",
+    opacity: 1,
+}
+
+export const App = () => {
+    const [isVisible, setIsVisible] = React.useState(false)
+    const isReducedMotion = useReducedMotion()
+    const transition = isReducedMotion ? { type: false } : { duration: 1 }
+    const variants = {
+        visible: { opacity: 1, transition },
+        hidden: { opacity: 0, transition },
+    }
+
+    React.useEffect(() => {
+        setTimeout(() => setIsVisible(!isVisible), 1500)
+    })
```

Comment:
Want to pass an empty array here? Otherwise this timeout will be set on each render, unclear if that's intended or not.

Reply:
yeah it's intended

Reply:
could pass isVisible